chromadb/test/property/test_embeddings.py

```diff
@@ -197,8 +212,14 @@ def upsert_embeddings(self, record_set: strategies.RecordSet) -> None:
 
     @invariant()
     def count(self) -> None:
+        n = invariants.get_n_items_from_record_set_state(self.record_set_state)
+        if n == 0:
+            assert self.collection.count() == 0
+            return
+
         invariants.count(
-            self.collection, cast(strategies.RecordSet, self.record_set_state)
+            self.collection,
+            cast(strategies.RecordSet, self.record_set_state),
```

Comment:
Shouldn't `count` just take the new `RecordSetState` type instead?

Reply:
make a note